### PR TITLE
Modal popup + adding likes to user post

### LINKED ISSUE #57: Modal popup for user posts

### PR DESCRIPTION
**Pull request by valmchl**

### CODE REVIEW

File: Flora-Fauna-Frontend/src/components/UserCards/UserCards.jsx [hunk 1]
```diff
@@ -1,11 +1,55 @@
...
+
+  const { source, title, desc, post } = props
+
+  const [visible, setVisible] = useState(false);
+  const [modalContent, setModalContent] = useState([]);
+  const modalHandler = () => setVisible(true);
+  
+  const closeHandler = () => {
+    setVisible(false);
+    console.log("closed");
```
Comment (LifeBringer) [inline Flora-Fauna-Frontend/src/components/UserCards/UserCards.jsx]: Remove/Comment out

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See comment

File: Flora-Fauna-Frontend/src/components/UserCards/UserCards.jsx (hunk 1 above)
Comment (valmchl) [reply Flora-Fauna-Frontend/src/components/UserCards/UserCards.jsx]: When I tried it, the modals would only open and close one time, then never again.

**Review by valmchl — COMMENTED** [review]
Removed console.log

File: Flora-Fauna-Frontend/src/components/UserCards/UserCards.jsx (hunk 1 above)
Comment (LifeBringer) [reply Flora-Fauna-Frontend/src/components/UserCards/UserCards.jsx]: That doesn't sound right... just delete the console.log

Comment (valmchl) [reply Flora-Fauna-Frontend/src/components/UserCards/UserCards.jsx]: Yes, I removed it

**Review by LifeBringer — APPROVED** [review]

**Pull request merged**